ttbd/ui/static/js/main.js

```diff
@@ -261,6 +261,71 @@ async function js_alloc_guest_remove(allocid, selector_id) {
     window.location.reload()
 }
 
+/*
+* Add a tunnel from form data
+*
+* @param {targetid} str -> target id to which you want to flash
+*
```

Comment:
probably we could add the other param to the docs

Reply:
smth like `
```
* @param {input_field_id} str -> element tag id for the input that has the....
```
And minor stuff but also fix the comment on targetid 
```
target id to which you want to add a tunnel to
```
or smth like that.